ciao-controller/legacy_api.go

```diff
@@ -270,43 +281,43 @@ func listServerDetailsFlavors(c *controller, w http.ResponseWriter, r *http.Requ
 
 func legacyComputeRoutes(ctl *controller, r *mux.Router) *mux.Router {
 	r.Handle("/v2.1/{tenant}/servers/action",
-		legacyAPIHandler{ctl, tenantServersAction}).Methods("POST")
+		legacyAPIHandler{ctl, tenantServersAction, false}).Methods("POST")
 
 	r.Handle("/v2.1/flavors/{flavor}/servers/detail",
-		legacyAPIHandler{ctl, listServerDetailsFlavors}).Methods("GET")
+		legacyAPIHandler{ctl, listServerDetailsFlavors, true}).Methods("GET")
```

Comment:
verify.sh is failing for me.  I see

```
Waiting for instance to boot
F0228 16:25:17.518129   13880 instance.go:767] ciao-cli FATAL: HTTP Error [401] for [GET https://singlevm:8774/v2.1/flavors/38acdccd-506b-4cad-acc4-ee092d898a2f/servers/detail]: Unauthorized
```

Should this one be false perhaps or does verify.sh need updating?